devel/zthread/Portfile

```diff
@@ -0,0 +1,103 @@
+license             MIT
+maintainers         nomaintainer
+                    #  v Note: No https site available
+homepage            http://${name}.sourceforge.net
+description         object-oriented threading and synchronization \
+                    library
+long_description    ZThread is an advanced ${description}, implemented \
+                    in C++. It provides an excellent and powerful \
+                    abstraction from native threads. It includes \
+                    interruptible thread objects and several other \
+                    synchronization control objects and lets you \
+                    safely terminate threads without the complications \
+                    of using cancellation handlers or other similar \
+                    constructs.
+
+master_sites        sourceforge:project/${name}/ZThread/${version}
+distname            ZThread-${version}
+checksums           rmd160  8419afad91ea8db1fff7ce3bf7784329839dd2e8 \
+                    sha256  950908b7473ac10abb046bd1d75acb5934344e302db38c2225b7a90bd1eda854 \
+                    size    412527
+
+patchfiles-append   dynamic_lookup-11.patch
+post-patch {
+    reinplace -E {s/(aclocal|automake)-\${am__api_version}/\1/} \
+        ${worksrcpath}/configure
+
+    # These regexes are similar to bugfixes from the Debian package:
+    # "020_fix-g++4.4.patch" and "050_fix-ftbfs-with-g++-4.7.patch"
+    foreach re [list \
+        {/void createScope.*long ms/s/void/bool/} \
+        {/if.*isDisabled/s/(isDisabled)/LockHolder<LockType>::\1/} \
+        {/LockingPolicy.*Scope.*extract/s/(extract)/this->\1/} \
+    ] {
+        reinplace -E $re ${worksrcpath}/include/zthread/Guard.h
+    }
+    # These regexes are similar to bugfixes from the Debian package:
+    # "020_fix-g++4.4.patch"
+    foreach re [list \
+        {/waiter(Arrived|Departed).*self/s/(waiter(Arrived|Departed))/MutexImpl<List, Behavior>::\1/} \
+        {/owner(Acquired|Released).*(self|impl)/s/(owner(Acquired|Released))/MutexImpl<List, Behavior>::\1/} \
+    ] {
+        reinplace -E $re ${worksrcpath}/src/MutexImpl.h
+    }
+
+    # Incorporate bugfix from Debian package:
+    # "030_executor-remove-thread-bug"
+    reinplace -E \
+        {/remove.*threads.*current/s/(std::remove.*);/_threads.erase(\1);/} \
+        ${worksrcpath}/src/PoolExecutor.cxx
+
+    # Incorporate bugfix from Debian package: "080_wrong-parameter-type"
+    reinplace {/virtual bool add/s/T item/const T\& item/} \
+        ${worksrcpath}/include/${name}/BlockingQueue.h
+
+    # Add missing DESTDIR to some of the install paths
+    foreach re [list \
+        {/mkinstalldirs.*(bin|data)dir/s/(\$.(bin|data)dir)/\$(DESTDIR)\1/} \
+        {/INSTALL_PROGRAM.*zthread-config/s/(\$.bindir)/\$(DESTDIR)\1/} \
+        {/INSTALL_DATA.*datadir.*aclocal/s/(\$.datadir)/\$(DESTDIR)\1/} \
+    ] {
+        reinplace -E $re {*}[glob -directory ${worksrcpath} Makefile.*]
+    }
+}
+
+configure.args-append   --prefix=${prefix}
```

Comment:
`--prefix=${prefix}` is already in `configure.pre_args`.

Reply:
Removed. 👍